docs/fsharp/language-reference/records.md

```diff
@@ -14,10 +14,7 @@ ms.assetid: 3a3701ea-4308-4fa1-9b5c-b955c470f17a
 
 # Records
 
-> [!NOTE]
-> The following article does not cover using the `struct` attribute for Records yet, which is an F# 4.1 feature.  It will be documented here.
-
-Records represent simple aggregates of named values, optionally with members.
+Records represent simple aggregates of named values, optionally with members.  Starting with F# 4.1, they can either be structs or reference types.  They are reference types by default.
```

Comment:
In the union section you added ``struct-attribute`` to the syntax.  Here we have ``attributes``.  I don't mind which - the second is easier and comprehensive enough - but we should be consistent.